horace_core/sqw/@Experiment/Experiment.m

```diff
@@ -2,9 +2,9 @@
     %EXPERIMENT Container object for all data describing the Experiment
     
     properties(Access=private)
-        instruments_ = IX_inst();
+        instruments_ = IX_inst.empty;
         detector_arrays_ = []
-        samples_ = IX_samp();
+        samples_ = IX_samp.empty;
         expdata_ = IX_experiment();
     end
```

Comment:
This form (with empty arrays) was causing problem with previous code. Need check if the problem had been fixed when failure due to old herbert is resolved

Reply:
These have now been overridden by empty cell arrays as described in the previous reply.